components/ProgressCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import '../scss/progressCard.scss'
+
+type Props = {
+  progressCount: number
+}
+
+const ProgressCard: React.FC<Props> = ({ progressCount }) => {
+  return (
+    <div className="progress-card__container d-flex card shadow-sm mt-3 bg-primary text-white p-2">
+      <div className="card-body">
+        <div className="progress-card__progress-bar d-flex justify-content-center align-items-center rounded-circle">
+          <span className="font-weight-bold">{progressCount}%</span>
+        </div>
+        <h4 className="progress-card__title mt-3">
```

Comment:
Since it's not really a title you might:
`<p class="h4 mt-3 progress-card__title">You're off to a great start!</p>` (same result)